slash/react/src/Card/Card.tsx

```diff
@@ -0,0 +1,35 @@
+import { ComponentProps } from "react";
+import { getComponentClassName } from "../utilities";
+import "@axa-fr/design-system-slash-css/dist/Card/Card.scss";
+import { Svg } from "../Svg";
+
+type CardProps = ComponentProps<"button"> & {
+  mode?: "vertical" | "horizontal";
+  /** icon url, for exemple from "@material-symbols" :
+   *
+   *  import villaIcon from "@material-symbols/svg-400/outlined/villa.svg"; */
+  icon?: string;
+  classModifier?: string;
+};
+
+export const Card = ({
+  children,
+  className,
+  classModifier,
+  mode = "vertical",
+  icon,
+  ...otherProps
+}: CardProps) => {
+  const componentClassName = getComponentClassName(
+    "af-card",
+    [classModifier, mode].join(" "),
+    className,
+  );
+
+  return (
+    <button type="button" className={componentClassName} {...otherProps}>
+      {icon && <Svg width="48" height="48" src={icon} />}
```

Comment:
```suggestion
      {icon ? <Svg width="48" height="48" src={icon} /> : null}
```

Reply:
Pourquoi forcer le null ?

Reply:
C'est une precaution qui peut, dans certains rare cas etre utile : https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Ca permet aussi d'eviter d'envoyer du garbage dans le jsx dom de react

Reply:
sinon au peut aussi faire 
```suggestion
      {Boolean(icon) && <Svg width="48" height="48" src={icon} />}
```